src/core/components/RootNavLayout.tsx

```diff
@@ -1,11 +1,15 @@
 import { Outlet } from 'react-router-dom';
 import { NavMenu } from './NavMenu';
+import './RootNavLayout.css';
```

Comment:
Please make a new folder for the RootNavLayout component and move there the both files (.tsx and css). Please also add the index.ts and make a named re-export for the component, that should simplify the import

Reply:
Done!